src/plugins/editor/editor.tsx

```diff
@@ -76,14 +82,21 @@ export class UIPEditor extends UIPPluginPanel {
     return CodeJar(this.$code, UIPEditor.highlight, {tab: '\t'});
   }
 
+  public get isJsReadonly(): boolean {
```

Comment:
Just a reminder we have planned readonly mode before current request. It does not limited to JS Editor or relates to isolation. We just decided to not implement manual readonly controls for now and implement just automatic part. (Btw I'm not restrictive, in case we want we can also implement manual part as well) 
```suggestion
  public get isReadonly(): boolean {
```